modular_robot_simulation/revolve2/modular_robot_simulation/build_multi_body_systems/concrete_builder/_active_hinge_builder.py

```diff
@@ -0,0 +1,134 @@
+from pyrr import Quaternion, Vector3
+
+from revolve2.modular_robot.body.common import ActiveHinge
+from revolve2.simulation.scene import AABB, JointHinge, Pose, RigidBody
+from revolve2.simulation.scene.geometry import GeometryBox
+
+from ..._modular_robot_active_hinge_key import ModularRobotActiveHingeKey
+from .._builder import Builder
+from .._builder_params import BuilderParams
+from .._convert_color import convert_color
+from .._queue_item import QueueItem
+
+
+class ActiveHingeBuilder(Builder):
+    """A Builder for Hinges."""
+
+    def build(self, module: ActiveHinge, builder_params: BuilderParams) -> list[QueueItem]:  # type: ignore
```

Comment:
If you must really use type: ignore, you should always explain why and what you are ignoring. However, it's probably possible to do without?